### PR TITLE
fix: change rgb color codes to css variables

### PR DESCRIPTION
**Pull request by jainpawan21**
Issue Number: #2451 

## PR Details

Changed hard coded RGB color codes to CSS variables

## PR Checklist
- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-design-system/src/style/css-variables.scss [hunk 1]
```diff
@@ -48,6 +48,8 @@
   --panel-background: var(--white);
   --panel-hover-background: var(--black5);
 
+  --mui-toggle-track-background :#e3ddf6;
```
Comment (yuval-hazaz) [inline packages/amplication-design-system/src/style/css-variables.scss:51]:
```suggestion
  --toggle-track-background :#e3ddf6;
```

File: packages/amplication-design-system/src/components/Toggle/Toggle.scss [hunk 2]
```diff
@@ -66,7 +66,7 @@ $thumb-position-offset: -6px;
         width: $track-width;
         height: $track-height;
         position: relative;
-        background: #e3ddf6;
+        background: var(--mui-toggle-track-background);
```
Comment (yuval-hazaz) [inline packages/amplication-design-system/src/components/Toggle/Toggle.scss:69]:
```suggestion
        background: var(--toggle-track-background);
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

**Comment by yuval-hazaz** [comment]
@jainpawan21 thanks for your contributions! it is very helpful and you are doing a great job
in order to make your contributions easier and to show our appreciation, I would be happy to add you to the @amplication/committers team. That will allow you to work on the main fork directly.

**Comment by jainpawan21** [comment]
> @jainpawan21 thanks for your contributions! it is very helpful and you are doing a great job in order to make your contributions easier and to show our appreciation, I would be happy to add you to the @amplication/committers team. That will allow you to work on the main fork directly.

Thank You @yuval-hazaz for your kind words.
I will be happy to be part of the **committers** team.

**Review by yuval-hazaz — APPROVED** [review]

**Review by Harry-Hopkinson — APPROVED** [review]

**Review by tupe12334 — APPROVED** [review]

**Pull request merged**